interface/modules/custom_modules/oe-module-claimrev-connect/src/EraPage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Modules\ClaimRevConnector;
+
+use OpenEMR\Modules\ClaimRevConnector\EraSearch;
+
+class EraPage
+{
+    public static function searchEras($postData)
+    {
+        $startDate = $_POST['startDate'];
+        $endDate = $_POST['endDate'];
+        $fileStatus = $_POST['downloadStatus'];
```

Comment:
pretty sure you meant above 3 lines to instead look like:
```
        $startDate = $postData['startDate'];
        $endDate = $postData['endDate'];
        $fileStatus = $postData['downloadStatus'];
```

Reply:
fixed